plugins/lang/admincmd.txt

```diff
@@ -91,10 +123,10 @@ PERM = für immer
 CLIENT_BANNED = Spieler "%s" gebannt
```

Comment:
FOR_MIN = für %d Minuten

Reply:
There's no param for %d in the .sma file.

Reply:
I thought at line 89? 
In the other languages you changed %s to %d?

Reply:
I didn't change any %s to %d.

Reply:
Am I blind? 

I'm talking about lines 
- 89 (in your version line 121)

there is "%d"

and there are "%s":
- 12
- 164 (in your version line 196)
- 239 (in your version line 271)
- 314 (in your version line 346)
- 389 (in your version line 421)
- 464 (in your version line 496)
- 539 (in your version line 571)
- 614 (in your version line 646)
- 681 (in your version line 721)
- 764  (in your version line 796)
and so on...

But you haven't change the .sma in that point, have you?

Reply:
You are not blind. `FOR_MIN` takes now an integer, so it should be `%d`.

Reply:
I see, I thought you meant line 89 in my version. I'll do the changes soon.